### PR TITLE
[hi] localize microservices.md

### PR DESCRIPTION
**Pull request by SahityaGupta25**
Signed-off-by: SahityaGupta25 <99082422+SahityaGupta25@users.noreply.github.com>
email - vasuguptadevops@gmail.com

### CODE REVIEW

File: content/hi/microservices.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+| Title                                    | Status    | Category |
+| ---------------------------------------- | --------- | -------- |
+| [Microservices](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) | Completed | Concept  |
```
Comment (anubha-v-ardhan) [inline content/hi/microservices.md:3]:
```suggestion
---
title: माइक्रोसर्विसेज़ (Microservices)
status: Completed
category: अवधारणा
---
```

File: content/hi/microservices.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+# यह क्या है
+
+```
+माइक्रोसर्विसेज अनुप्रयोग विकास के लिए एक आधुनिक दृष्टिकोण है जो क्लाउड नेटिव तकनीकों का लाभ उठाता है। जबकि आधुनिक एप्लिकेशन, जैसे नेटफ्लिक्स, एक एकल ऐप प्रतीत होते हैं, वे वास्तव में छोटी सेवाओं का एक संग्रह हैं - सभी एक साथ मिलकर काम कर रहे हैं। उदाहरण के लिए, एक एकल पृष्ठ जो आपको वीडियो तक पहुंचने, खोजने और पूर्वावलोकन करने की अनुमति देता है, संभवतः छोटी सेवाओं द्वारा संचालित होता है जो प्रत्येक इसके एक पहलू को संभालते हैं (जैसे आपके ब्राउज़र में खोज, प्रमाणीकरण और पूर्वावलोकन चलाना)। संक्षेप में, माइक्रोसर्विसेज एक एप्लिकेशन आर्किटेक्चर पैटर्न को संदर्भित करता है जो आमतौर पर  मोनोलिथिक एप्लिकेशन के विपरीत होता है।
+
+```
+
+#समस्या यह संबोधित करता है
```
Comment (anubha-v-ardhan) [inline content/hi/microservices.md:12]:
```suggestion
# समस्या
```

File: content/hi/microservices.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+[Micro Services](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) अखंड अनुप्रयोगों द्वारा उत्पन्न चुनौतियों की प्रतिक्रिया है। आम तौर पर, किसी एप्लिकेशन के विभिन्न हिस्सों को अलग-अलग स्केल करने की आवश्यकता होगी। एक ऑनलाइन स्टोर, उदाहरण के लिए, चेकआउट की तुलना में अधिक उत्पाद दृश्य होने वाला है। इसका मतलब है कि आपको चेकआउट की तुलना में चल रहे उत्पाद दृश्य कार्यक्षमता की अधिक प्रतियों की आवश्यकता होगी। एक अखंड अनुप्रयोग में, तर्क के उन बिट्स को अलग से तैनात नहीं किया जा सकता है। यदि आप व्यक्तिगत रूप से उत्पाद की कार्यक्षमता को माप नहीं सकते हैं, तो आपको पूरे ऐप को उन सभी घटकों के साथ डुप्लिकेट करना होगा जिनकी आपको आवश्यकता नहीं है - संसाधनों का एक अक्षम उपयोग। मोनोलिथिक एप्लिकेशन भी डेवलपर्स के लिए डिजाइन के नुकसान के आगे झुकना आसान बनाते हैं। चूंकि सभी कोड एक ही स्थान पर हैं, इसलिए उस कोड को कसकर युग्मित करना आसान है और चिंताओं को अलग करने के सिद्धांत को लागू करना कठिन है। मोनोलिथ को अक्सर डेवलपर्स को उत्पादक होने से पहले पूरे कोडबेस को समझने की आवश्यकता होती है।
+
+# यह कैसे मदद करता है
```
Comment (anubha-v-ardhan) [inline content/hi/microservices.md:16]:
```suggestion
# समाधान
```

File: content/hi/microservices.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+| Title                                    | Status    | Category |
+| ---------------------------------------- | --------- | -------- |
+| [Microservices](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) | Completed | Concept  |
+
+# यह क्या है
+
+```
+माइक्रोसर्विसेज अनुप्रयोग विकास के लिए एक आधुनिक दृष्टिकोण है जो क्लाउड नेटिव तकनीकों का लाभ उठाता है। जबकि आधुनिक एप्लिकेशन, जैसे नेटफ्लिक्स, एक एकल ऐप प्रतीत होते हैं, वे वास्तव में छोटी सेवाओं का एक संग्रह हैं - सभी एक साथ मिलकर काम कर रहे हैं। उदाहरण के लिए, एक एकल पृष्ठ जो आपको वीडियो तक पहुंचने, खोजने और पूर्वावलोकन करने की अनुमति देता है, संभवतः छोटी सेवाओं द्वारा संचालित होता है जो प्रत्येक इसके एक पहलू को संभालते हैं (जैसे आपके ब्राउज़र में खोज, प्रमाणीकरण और पूर्वावलोकन चलाना)। संक्षेप में, माइक्रोसर्विसेज एक एप्लिकेशन आर्किटेक्चर पैटर्न को संदर्भित करता है जो आमतौर पर  मोनोलिथिक एप्लिकेशन के विपरीत होता है।
+
+```
```
Comment (anubha-v-ardhan) [inline content/hi/microservices.md:10]:
```suggestion
माइक्रोसर्विसेज एप्लीकेशन विकास के लिए एक आधुनिक दृष्टिकोण है जो क्लाउड नेटिव तकनीकों का लाभ उठाता है। जहाँ आधुनिक एप्लिकेशन, जैसे नेटफ्लिक्स (Netflix), एक एकल ऐप प्रतीत होते हैं, वे वास्तव में छोटी सेवाओं का एक संग्रह हैं - सभी एक साथ मिलकर काम कर रहे हैं। उदाहरण के लिए, एक एकल पृष्ठ जो आपको वीडियो तक पहुंचने, खोजने और पूर्वावलोकन करने की अनुमति देता है, संभवतः छोटी सेवाओं द्वारा संचालित होता है जो प्रत्येक इसके एक पहलू को संभालते हैं (जैसे आपके ब्राउज़र में खोज, प्रमाणीकरण और पूर्वावलोकन चलाना)। संक्षेप में, माइक्रोसर्विसेज एक एप्लिकेशन आर्किटेक्चर पैटर्न को संदर्भित करता है जो आमतौर पर  मोनोलिथिक एप्लिकेशन के विपरीत होता है।
```

File: content/hi/microservices.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+#समस्या यह संबोधित करता है
+
+[Micro Services](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) अखंड अनुप्रयोगों द्वारा उत्पन्न चुनौतियों की प्रतिक्रिया है। आम तौर पर, किसी एप्लिकेशन के विभिन्न हिस्सों को अलग-अलग स्केल करने की आवश्यकता होगी। एक ऑनलाइन स्टोर, उदाहरण के लिए, चेकआउट की तुलना में अधिक उत्पाद दृश्य होने वाला है। इसका मतलब है कि आपको चेकआउट की तुलना में चल रहे उत्पाद दृश्य कार्यक्षमता की अधिक प्रतियों की आवश्यकता होगी। एक अखंड अनुप्रयोग में, तर्क के उन बिट्स को अलग से तैनात नहीं किया जा सकता है। यदि आप व्यक्तिगत रूप से उत्पाद की कार्यक्षमता को माप नहीं सकते हैं, तो आपको पूरे ऐप को उन सभी घटकों के साथ डुप्लिकेट करना होगा जिनकी आपको आवश्यकता नहीं है - संसाधनों का एक अक्षम उपयोग। मोनोलिथिक एप्लिकेशन भी डेवलपर्स के लिए डिजाइन के नुकसान के आगे झुकना आसान बनाते हैं। चूंकि सभी कोड एक ही स्थान पर हैं, इसलिए उस कोड को कसकर युग्मित करना आसान है और चिंताओं को अलग करने के सिद्धांत को लागू करना कठिन है। मोनोलिथ को अक्सर डेवलपर्स को उत्पादक होने से पहले पूरे कोडबेस को समझने की आवश्यकता होती है।
```
Comment (anubha-v-ardhan) [inline content/hi/microservices.md:14]:
```suggestion
माइक्रोसर्विस अखंड एप्लीकेशनों द्वारा उत्पन्न चुनौतियों की प्रतिक्रिया है। आम तौर पर, किसी एप्लिकेशन के विभिन्न हिस्सों को अलग-अलग [स्केल](/scalability/) करने की आवश्यकता होती है। उदाहरण के लिए, एक ऑनलाइन स्टोर, चेकआउट की तुलना में अधिक उत्पाद दर्शाता है। इसका मतलब है कि आपको चेकआउट की तुलना में चल रहे उत्पाद दृश्य कार्यक्षमता की अधिक प्रतियों की आवश्यकता होगी। एक अखंड एप्लीकेशन में, उन बिट्स को अलग से तैनात नहीं किया जा सकता है। यदि आप व्यक्तिगत रूप से उत्पाद की कार्यक्षमता को माप नहीं सकते हैं, तो आपको पूरे ऐप को उन सभी घटकों के साथ डुप्लिकेट करना होगा जिनकी आपको आवश्यकता नहीं है - संसाधनों का एक अक्षम उपयोग। मोनोलिथिक एप्लिकेशन भी डेवलपर्स के लिए डिजाइन के नुकसान को आसान बनाते हैं। चूंकि सभी कोड एक ही स्थान पर हैं, इसलिए उस कोड को [युग्मित (tightly-coupled)](/tightly-coupled-architectures/) करना आसान है और चिंताओं को अलग करने के सिद्धांत को लागू करना कठिन है। मोनोलिथ में अक्सर डेवलपर्स को उत्पादक होने से पहले पूरे कोडबेस को समझने की आवश्यकता होती है।
```

File: content/hi/microservices.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+# यह कैसे मदद करता है
+
+```
+विभिन्न माइक्रोसर्विसेज में कार्यक्षमता को अलग करने से उन्हें स्वतंत्र रूप से तैनात, अपडेट और स्केल करना आसान हो जाता है। अलग-अलग टीमों को बड़े एप्लिकेशन के अपने छोटे हिस्से पर ध्यान केंद्रित करने की अनुमति देकर आप उनके लिए बाकी संगठन को नकारात्मक रूप से प्रभावित किए बिना अपने ऐप्स पर काम करना आसान बनाते हैं। जबकि माइक्रोसर्विसेज कई समस्याओं का समाधान करते हैं, वे परिचालन ओवरहेड भी बनाते हैं - वे चीजें जिन्हें आपको तैनात करने और परिमाण या अधिक के क्रम में वृद्धि का ट्रैक रखने की आवश्यकता होती है। कई क्लाउड-देशी तकनीकों का उद्देश्य माइक्रोसर्विसेज को तैनात और प्रबंधित करना आसान बनाना है।
+```
```
Comment (anubha-v-ardhan) [inline content/hi/microservices.md:20]:
```suggestion
विभिन्न माइक्रोसर्विसेज में कार्यक्षमता को अलग करने से उन्हें स्वतंत्र रूप से तैनात, अपडेट और स्केल करना आसान हो जाता है। अलग-अलग टीमों को बड़े एप्लिकेशन के अपने छोटे हिस्से पर ध्यान केंद्रित करने की अनुमति देकर आप उनके लिए बाकी संगठन को नकारात्मक रूप से प्रभावित किए बिना अपने ऐप्स पर काम करना आसान बनाते हैं। माइक्रोसर्विसेज कई समस्याओं का समाधान करते हैं पर वे परिचालन ओवरहेड भी बनाते हैं - वे चीजें जिन्हें आपको तैनात करने और परिमाण या अधिक के क्रम में वृद्धि का ट्रैक रखने की आवश्यकता होती है। कई [क्लाउड-नेटिव तकनीकों](/cloud-native-tech/) का उद्देश्य माइक्रोसर्विसेज को तैनात और प्रबंधित करना आसान बनाना है।

```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you @SahityaGupta25.
Please go though these suggested changes and commit them if you agree.

**Comment by Garima-Negi** [comment]
@SahityaGupta25 any updates on the changes requested?

**Comment by SahityaGupta25** [comment]
@Garima-Negi No

**Comment by anubha-v-ardhan** [comment]
Hello @SahityaGupta25 Are you still working on this PR? Any updates about the requested changes

**Comment by Garima-Negi** [comment]
closing this PR due to inactivity

**Pull request closed without merging**